Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenStick.cs

```diff
@@ -13,47 +13,40 @@ public class OnScreenStick : OnScreenControl, IPointerDownHandler, IPointerUpHan
     {
         private void Start()
         {
-            m_StartPos = transform.position;
+            m_StartPos = (transform as RectTransform).anchoredPosition;
+            Canvas canvas = GetComponentInParent<Canvas>();
+            if (canvas)
+                m_Camera = canvas.worldCamera;
```

Comment:
Not sure caching this here is safe (especially in the current code that only looks things up once at start). Can we get this from `PointerEventData`.